### PR TITLE
Example of how to set slow logs dynamically per-index

### PR DESCRIPTION
**Pull request by luizgpsantos**

### CODE REVIEW

**Comment by rjernst** [comment]
@jrodewig Can you review this please?

File: docs/reference/index-modules/slowlog.asciidoc
Comment (jrodewig) [inline docs/reference/index-modules/slowlog.asciidoc]: Minor edit to make the `_settings` API example more explicit.
```suggestion
All of the above settings are _dynamic_ and can be set for each index using the
<<indices-update-settings, update indices settings>> API. For example: 
```

File: docs/reference/index-modules/slowlog.asciidoc
Comment (jrodewig) [inline docs/reference/index-modules/slowlog.asciidoc]:
```suggestion
All of the above settings are _dynamic_ and can be set for each index using the
<<indices-update-settings, update indices settings>> API. For example: 
```

File: docs/reference/index-modules/slowlog.asciidoc
Comment (jrodewig) [inline docs/reference/index-modules/slowlog.asciidoc]: This should fix the CI errors.
```suggestion
// CONSOLE
// TEST[setup:twitter]
```

File: docs/reference/index-modules/slowlog.asciidoc
Comment (jrodewig) [inline docs/reference/index-modules/slowlog.asciidoc]:
```suggestion
// CONSOLE
// TEST[setup:twitter]
```

**Review by jrodewig — CHANGES_REQUESTED** [review]
Thanks for the contribution @luizgpsantos!

I made a few suggestions that should get this to pass the CI. As soon as those are fixed, I can help merge and backport.

**Review by jrodewig — APPROVED** [review]
LGTM. I'll merge this and backport as needed once CI passes. Thanks again @luizgpsantos!

**Comment by luizgpsantos** [comment]
Thanks @jrodewig!

**Pull request merged**
